apps/sched/interface.html

```diff
@@ -263,12 +263,25 @@
   alarms.push(timer);
 }
 
+function readStorageIfExists(filename, cb) {
+  Puck.write(`\x10(function() {
+    Bluetooth.print(JSON.stringify(require("Storage").list("${filename}").sort()));
```

Comment:
If the filename is a substring we'll get a match, e.g. `require("Storage").list("rep.j")` will match `rep.json`, which might not be what we want - perhaps we could do a `require("Storage").read("rep.j") != undefined` ?

Or if we're promoting this into `Util`, we could modify [reading of files](https://github.com/espruino/EspruinoAppLoaderCore/blob/86819223f43a892622a8d5f1f430a2a1fc57031d/js/comms.js#L478-L502) to permit returning `undefined` if the file doesn't exist? (maybe I'm over thinking here though)

Reply:
Should be fixed and yes, having that in util would be
prefered.

Reply:
Nice - I could sort out a follow up to pop it into util and sort out the `rep` app